.babelrc

```diff
@@ -1,3 +1,12 @@
 {
-    "presets": ["es2015"]
+    "presets": [
```

Comment:
Can you change the `.babelrc` to be two-space indented to match the other JSON config files? (I know it was 4-spaced before... but may as well fix while the file's being rewritten anyway)